crates/rust-analyzer/tests/heavy_tests/support.rs

```diff
@@ -212,7 +212,7 @@ impl Server {
                     ProgressParams {
                         token: lsp_types::ProgressToken::String(ref token),
                         value: ProgressParamsValue::WorkDone(WorkDoneProgress::End(_)),
-                    } if token == "rustAnalyzer/startup" => true,
+                    } if token == "rustAnalyzer/rootsScanned" => true,
```

Comment:
I haven't looked over this in detail, but we have a `rustAnalyzer/roots scanned` in `main_loop.rs` and `heavy_tests/support.rs` and `rootsScanned` here. Is this okay?

Reply:
No, but that should was fixed as a part of #5046